examples/pipelines/asl_nistats_processing.py

```diff
@@ -0,0 +1,170 @@
+    paradigm_fn = op.join(archives, subject, 'events', 'acquisition1', 'paradigm_' + subject + '_ASL_overlap_paradigm-version-2b.csv')
+    paradigm=experimental_paradigm.paradigm_from_csv(paradigm_fn)
+    dm2 =nt.make_design_matrix_asl(paradigm, n_scans, t_r, hrf_length=32., oversampling=16, hrf_model='canonical', prf_model='physio', drift_model='Cosine', drift_order=7, hf_cut=hfcut, mvt_file=mvt_file[1])
+    names = dm1.columns.values
+
+    # Contrasts
+    contrasts = {}
+    contrast_matrix = np.eye(len(names))
+    for i in np.arange(0, len(names)):
+        contrasts[names[i]] = contrast_matrix[i]
+    contrasts = OrderedDict({'motor_audio_R-L': contrasts['motor_audio_right'] - contrasts['motor_audio_left'], 'motor_audio_L-R': contrasts['motor_audio_left'] - contrasts['motor_audio_right'], 'visual': contrasts['visual'], 'perfusion_motor_audio_R-L': contrasts['perfusion_motor_audio_right'] - contrasts['perfusion_motor_audio_left'], 'perfusion_motor_audio_L-R': contrasts['perfusion_motor_audio_left'] - contrasts['perfusion_motor_audio_right'], 'perfusion_visual': contrasts['perfusion_visual'], 'perfusion_baseline': contrasts['perfusion_baseline']})
+
+    # First level GLM
+    data_img1, data_img2, anat_img = nb.load(data_fn1), nb.load(data_fn2), nb.load(anat_fn)
+    if subject[:2] in ("MB", "MP"):
+        data_img1 = nb.Nifti1Image(data_img1.get_data(), affine=affine)
+        data_img2 = nb.Nifti1Image(data_img2.get_data(), affine=affine)
+
+    niimg1 = masker.inverse_transform(masker.transform(data_img1))
+    subject_data = nt.run_glm_nistats_contrasts(subject, output_dir, niimg1, dm1, contrasts, mask_img, noise_model='ols', nsession=1)
+    first_levels.append(subject_data)
+    niimg2 = masker.inverse_transform(masker.transform(data_img2))
+    subject_data = nt.run_glm_nistats_contrasts(subject, output_dir, niimg2, dm2, contrasts, mask_img, noise_model='ols', nsession=2)
+    first_levels.append(subject_data)
+
+
+
+# run second-level GLM
+
+output_dir = op.join(archives, 'results_glm_asl')
+if not op.exists(output_dir):
+    os.makedirs(output_dir)
+
+group_z_map = group_one_sample_t_test([subject_data["mask"] for subject_data in first_levels], [subject_data["effects_maps"] for subject_data in first_levels], first_levels[0]["contrasts"], output_dir)
+
+map_type = 'z'
+group_stats = []
+for contrast_id, file_name in group_z_map.iteritems():
+    zmap = nb.load(file_name)
+    if "perfusion_baseline"==contrast_id:
+        p = 0.0000001
+        img = zmap
+    elif 'perfusion' in contrast_id:
+        p = 0.3
+        img, zth = map_threshold(zmap, mask_img=mask_img, threshold=0.001, height_control='fdr', cluster_threshold=50)
+    else:
+        p = 0.3
+        img, zth = map_threshold(zmap, mask_img=mask_img, threshold=0.001, height_control='fdr', cluster_threshold=50)
+    plot_stat_map(zmap, title=contrast_id, display_mode='ortho', threshold=(zmap.get_data().max()*p))
```

Comment:
`ortho` is the defalut `display_mode` of `plot_stat_map`, so let's save same lines ;)